pkg/config/validation.go

```diff
@@ -149,3 +153,27 @@ func validatePodIntegrationOptions(c *configapi.Configuration) field.ErrorList {
 
 	return allErrs
 }
+
+func validateFairSharing(c *configapi.Configuration) field.ErrorList {
+	fs := c.FairSharing
+	if fs == nil {
+		return nil
+	}
+	var allErrs field.ErrorList
+	if !fs.Enable && len(fs.PreemptionStrategies) != 0 {
+		allErrs = append(allErrs, field.Invalid(fsPreemptionStrategiesPath, fs.PreemptionStrategies, "Must be empty when fair sharing is disabled"))
+	}
+	strategies := sets.New[configapi.PreemptionStrategy]()
+	validStrategies := []configapi.PreemptionStrategy{configapi.LessThanInitialShare, configapi.LessThanOrEqualToFinalShare}
+	for i, s := range fs.PreemptionStrategies {
+		path := fsPreemptionStrategiesPath.Index(i)
+		if slices.Index(validStrategies, s) == -1 {
```

Comment:
```suggestion
		if !slices.Contains(validStrategies, s) {
```

I understand that the `Index` and `Contains` have the same computation complexity, but the `Contains` is more direct, right?

Reply:
+1 on readability. Thanks!